From 1a9f0229ccd623a38399b8f4c450d59c22cc5f71 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 11 Jul 2023 17:16:50 +0200
Subject: [PATCH 0012/1141] ARM: dts: stm32: add sram node within
 stm32mp151.dtsi

Introduce the sram node in order to be used by drivers
requiring SRAM memory space.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: Iad2c37b1b40b107ac7655d0b8a561a3b6c329a2c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/319550
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/boot/dts/stm32mp151.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/stm32mp151.dtsi b/arch/arm/boot/dts/stm32mp151.dtsi
index c3ccc69ad23d..3c41c59cef26 100644
--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -153,6 +153,14 @@ soc {
 		interrupt-parent = <&intc>;
 		ranges;
 
+		sram4: sram@10050000 {
+			compatible = "mmio-sram";
+			reg = <0x10050000 0x10000>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0 0x10050000 0x10000>;
+		};
+
 		hsem: hwspinlock@4c000000 {
 			compatible = "st,stm32-hwspinlock";
 			#hwlock-cells = <2>;
-- 
2.39.2

